src/BulletSoftBody/btSoftBodyInternals.h

```diff
@@ -536,6 +536,14 @@ static SIMD_FORCE_INLINE bool bernsteinVFTest(const btSoftBody::Face* face, cons
     return coplanarAndInsideTest(k0, k1, k2, k3, face, node, dt);
 }
 
+template <class T>
+static SIMD_FORCE_INLINE void swap(T& a, T& b)
```

Comment:
there is a btSwap here, do you want a local version?
https://github.com/bulletphysics/bullet3/blob/master/src/LinearMath/btScalar.h#L642

Reply:
No, I will delete it. Thanks!